### PR TITLE
Add Spoofax v2.5.9

### PR DESCRIPTION
**Pull request by Baltoli**
Spoofax is a set of tools implementing a programming language
development workbench. This cask installs the stable Spoofax
distribution.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/spoofax.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'spoofax' do
+  version '2.5.9'
+  sha256 '62dc7c3a9052acaa3bdb8524279859b9c9a75d7b68a9202cf242aba6b2ad5860'
+
+  url 'https://artifacts.metaborg.org/service/local/repositories/releases/content/org/metaborg/org.metaborg.spoofax.eclipse.dist/2.5.9/org.metaborg.spoofax.eclipse.dist-2.5.9-macosx-x64-jre.tar.gz'
```
Comment (miccal) [inline Casks/spoofax.rb:5]: Please change this to:
```
url "https://artifacts.metaborg.org/service/local/repositories/releases/content/org/metaborg/org.metaborg.spoofax.eclipse.dist/#{version}/org.metaborg.spoofax.eclipse.dist-#{version}-macosx-x64-jre.tar.gz"
```
and also add an `appcast`:
```
appcast 'https://github.com/metaborg/spoofax/releases.atom'
```

Comment (Baltoli) [reply Casks/spoofax.rb]: Done - thanks!

Comment (miccal) [reply Casks/spoofax.rb]: Thank you, but the `appcast` stanza should be between the `url` and `name` stanzas.

Comment (Baltoli) [reply Casks/spoofax.rb]: Fixed that now - sorry for the trivial change!

**Pull request merged**